### PR TITLE
List of applications tested with S3 Server

### PR DESCRIPTION
**Pull request by nicolas2bert**

### CODE REVIEW

File: TESTED.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+### List of applications that have been tested with S3 Server
+
+- [Cyberduck](https://cyberduck.io/?l=en)
+- [Cloud Explorer](https://github.com/scality/S3/issues/277)
+- [AWS SDK](http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html)
+- [s3curl](https://github.com/rtdp/s3curl)
+- [s3cmd](http://s3tools.org/s3cmd)
+- [fog (Ruby)](http://fog.io/storage/)
+- [boto (python)](https://github.com/boto/boto)
```
Comment (DavidPineauScality) [inline TESTED.md]: should we reference separately boto2 and boto3 ? Or do you think it's irrelevant as they should be providing the same compatibility ?

**Comment by rahulreddy** [comment]
I think we should include config examples for the applications as well, as each application has it's own quirks for setup. This would make it easy for end users who are new to S3.

File: TESTED.md (hunk 1 above)
Comment (rahulreddy) [inline TESTED.md]: We should separate them (boto2 and boto3) and provide sample config for each of them. Both have different options for setup.

Comment (nicolas2bert) [inline TESTED.md]: Got it, thanks guys

**Comment by nicolas2bert** [comment]
@rahulreddy Yes that makes sense, I am going to do that.

File: TESTED.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#  Tested with S3
+
+### List of applications that have been tested with S3 Server
+
+- [Cyberduck](https://cyberduck.io/?l=en)
+- [Cloud Explorer](https://github.com/scality/S3/issues/277)
+- [AWS SDK](http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html)
```
Comment (LaurenSpiegel) [inline TESTED.md]: we should include both the java sdk and the javascript sdk.

**Pull request merged**